ringpop.go

```diff
@@ -531,7 +531,11 @@ func (rp *Ringpop) handleChanges(changes []swim.Change) {
 		}
 	}
 
+	start := time.Now()
 	rp.ring.AddRemoveServers(serversToAdd, serversToRemove)
+	treeConstructionTime := time.Now().Sub(start)
+	rp.statter.RecordTimer(rp.getStatKey("tree"), nil, treeConstructionTime)
```

Comment:
I think this is a `hashring` stat and not a ringpop stat so also add `ring` to the path as is done for `ring.server-added` `ring.server-removed` and `ring.changed` a couple of lines above.